projects/storefrontapp-e2e-cypress/cypress/e2e/vendor/cpq/download-proposal-document.e2e.flaky.cy.ts

```diff
@@ -0,0 +1,39 @@
+/*
```

Comment:
MAJOR: you can remove 'flaky' from file name as tests under vendor are not run in pipeline